tests/valid/elements.text

```diff
@@ -1092,6 +1093,7 @@ Unnumbered section
    *    Indent="5".  “The heaviest penalty for declining to rule is to
         be ruled by someone inferior to yourself.” Πολιτεία
 
+   *
```

Comment:
And now this is perhaps unfortunate. What we've uncovered is, I think, a decision to special case the rendering of nested lists without leading text that doesn't generalize to all possible uses of such things, and we're just toggling cross the edges of that earlier decision.

Has the previous side-effect of no leading marker been used in _this_ case previously? Is it ok to make this change (since getting the behavior of that side-effect is possible in better defined ways, I _think)?

@cabo - what do you think the right thing to do with this is?

Reply:
Here's a list of RFCs that has `ul` tag inside a `li` as first element:
```
rfc8668.xml
rfc8742.xml
rfc8811.xml
rfc8793.xml
rfc8927.xml
rfc8677.xml
rfc8667.xml
rfc8698.xml
rfc9195.xml
```
All these use `empty="true"` in parent `ul`, so no new `*` gets added but there's a bug in the suggested fix because it adds a new line. I'll fix that.

Reply:
> @cabo - what do you think the right thing to do with this is?

I don't think it is ever correct to add a new line that the author didn't want.
If it is needed, the author can put it in, but it never should be forced on them.

Reply:
I have pushed a fix to handle `empty="true"`.
See https://github.com/ietf-tools/xml2rfc/pull/1137/commits/11ef6ac0d304a1ee204ae3325e3d3ca700cfc546 and https://github.com/ietf-tools/xml2rfc/pull/1137/commits/fe595b1376d9359f27a3879b633d272d4d016f45

Reply:
@rjsparks & @cabo For context HTML output doesn't add a new line.
<img width="739" alt="Screenshot 2024-07-12 at 19 49 28" src="https://github.com/user-attachments/assets/512c9913-9702-4bb7-a8de-0edb00615d01">

Reply:
And HTML has the marker, so the text should to - that convinces me that (except for the newline that Carsten doesn't want) the current proposed change is right. 

So people looking at this later can see it in context, could you also paste the section of html analogous to  the `4.  4.1` change in the text in this thread?

Reply:
@rjsparks `ol` has a new line in HTML:
<img width="759" alt="Screenshot 2024-07-13 at 14 23 39" src="https://github.com/user-attachments/assets/c0909c39-96dc-49a0-bf0b-4c13bcc24ded">

Reply:
@rjsparks The above screenshot is from Firefox and that might not be showing the correct rending. See #1158